Dockerfile

```diff
@@ -3,8 +3,8 @@ FROM python:3-stretch
 RUN pip install uwsgi
 
 WORKDIR /app
-COPY requirements.txt /app/
-RUN pip install -r requirements.txt
```

Comment:
Well, you might need to copy requirements.txt *and* requirements dir with the other files to `/app/` here.